pkg/apis/apis/v1alpha1/types_apibinding.go

```diff
@@ -65,13 +65,32 @@ type APIBindingSpec struct {
 	// +kubebuilder:validation:Required
 	Reference ExportReference `json:"reference"`
 
-	// acceptedPermissionClaims records the permissions that are granted
-	// to the bound workspace.
-	// Access is granted on a GroupResource basis and can be filtered on objects by many different selectors.
+	// permissionClaims records the permissions that are granted to the bound workspace. Access is granted per
+	// GroupResource. Individual claims may be accepted or rejected.
```

Comment:
```suggestion
	// permissionClaims records the decision of the workspace owner about permission claims 
	// requested by the API service provider. The workspace owner can accept or reject them. In
	// former case the API service provider get the requested access to the workspace objects,
	// in the later case the API service provider will not. Access is granted per GroupResource.
```

Reply:
Update coming in next commit (I tweaked what you wrote a bit)